.gitignore

```diff
@@ -21,8 +21,9 @@ node_modules
 tmp
 dist/
 config/*
-!config/local.sh
-!config/env.docker.sh
+!config/.env.sh
+!config/.env.base.sh
+!config/.env.local.template
```

Comment:
Not sure why you want to use hidden filenames ?
why not simply:
`env.sh`, `env.base.sh` etc. ?

Reply:
because it is a common and wide pattern to use  dot file